src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Queries/SearchCategoryQueryHandler.cs

```diff
@@ -38,7 +38,13 @@ ISearchProvider searchProvider
 
         public virtual async Task<SearchCategoryResponse> Handle(SearchCategoryQuery request, CancellationToken cancellationToken)
         {
+            var terms = new List<string>();
+
             var store = await _storeService.GetByIdAsync(request.StoreId);
+            if (store != null)
```

Comment:
Can we throw in case storeId is specified and no store found?

Reply:
Done.